.github/ISSUE_TEMPLATE/custom-pull-request-template.md

```diff
@@ -0,0 +1,30 @@
+# Description
+
+Please include a summary of the changes and attch the related issue link. 
+
+Summary: 
+
+Issue Link: 
+
+# Date
+
+2023/9/18 21:22pm
```

Comment:
Do we need the Date here? I thought Github would automatically log the date :)

Reply:
Thats fair, I will remove the date section